.gcloudignore

```diff
@@ -0,0 +1 @@
+!target/*.jar
```

Comment:
Maybe add a comment explaining that we are explicitly excluding those files as they are included in .gotignore

Reply:
And that we need it to run the 'gcloud builds submit' command that builds a docker image and uploads it to the image repository (GCP)